### DISCUSSION TITLE
2.2-RC2

### DISCUSSION THREAD

**Discussion by Ghost-chu** [PBH-BTN/PeerBanHelper#16] [公告]
## 修复

1. 修复了 2.2-RC1 中 Transmission 模块在本地映像（Native Image）中报错崩溃的问题。

<hr /><em>This discussion was created from the release <a href='https://github.com/Ghost-chu/PeerBanHelper/releases/tag/2.2-RC2'>2.2-RC2</a>.</em>

**Comment by rihalvnium76** [comment]
好像native的日志输出的版本号没改，还是2.2-RC1。
